### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by geh2024** [aclap-dev/video-downloadhelper#704] [Help and Q&A]
### Your question

This site load a dummy video instead of the display file or gives the following error.
{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://indigowhitetv.com/videoshub/gwen-tennyson-has-a-lewd-dream",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [hls @ 0000021d8521a500] Unable to open key file https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080.key\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00000.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00001.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00002.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00003.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00004.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00005.ts\r\n[hls @ 0000021d8521a500] Error when loading first segment 'https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00000.ts'\r\n[in#0 @ 0000021d83878c40] Error opening input: Invalid data found when processing input\r\nError opening input file https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080.m3u8?Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9obHMyLnZpZGVvcy5zcHJvdXR2aWRlby5jb20vZmQ1Mjg1YzQxOGI1NTk5ODM4NTgwOGRmMmQ2OWVkMDEvZjI2M2M5MjNjNTkzMGFiNWY5MmMwMDI4NDI1YzA1YTQvKi5tM3U4P3Nlc3Npb25JRD0wYTM4NTlmNi0yNjY4LTQ0NDEtOTE4NS0wODZhZTZhZDY4ZmEiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE3MTY4MTg4ODR9fX1dfQ__&Signature=Xddda-xbZOiIehzAyAWnUzBhMwdLcsdsHUW3NnVHF4O272nh8-iHFI0VPqWw9-n4aTUHF00Ty45J60cKYNgdrRLncPtLnXoWkDMJUvFNstYu63tyDcJIWX5cyShBp1Gds~CgQ8Ns6NAVxn~mCsZ1Cxb4yK~p5AeLs0YArXmp~Gp07q99KWVmrxhTaaDHoTAz6UJznDHzVsErUk1FyC~ezsHJ-4Muy9mf9AEeeGG-m67gtrrgdocty8D5OR6PHq7mtORPOA594oE-WeOrxBMErZOHzz-ck9p4m-NpOWn8aLXNMssAsziVacRq-kTeweaLAB9vOwiFsyi9YUBb~y1p2A__&Key-Pair-Id=APKAIB5DGCGAQJ4GGIUQ&sessionID=0a3859f6-2668-4441-9185-086ae6ad68fa.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_cf0b6673-f855-45e8-ba7f-15ff62fda442",
    "downloadable_id": "downloadable_5843639359853216"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://indigowhitetv.com/videoshub/gwen-tennyson-has-a-lewd-dream",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [hls @ 0000021d8521a500] Unable to open key file https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080.key\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00000.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00001.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00002.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00003.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00004.ts\r\n[crypto @ 0000021d8528d0c0] Unable to open resource: https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00005.ts\r\n[hls @ 0000021d8521a500] Error when loading first segment 'https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080_00000.ts'\r\n[in#0 @ 0000021d83878c40] Error opening input: Invalid data found when processing input\r\nError opening input file https://hls2.videos.sproutvideo.com/fd5285c418b55998385808df2d69ed01/f263c923c5930ab5f92c0028425c05a4/video/1080.m3u8?Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9obHMyLnZpZGVvcy5zcHJvdXR2aWRlby5jb20vZmQ1Mjg1YzQxOGI1NTk5ODM4NTgwOGRmMmQ2OWVkMDEvZjI2M2M5MjNjNTkzMGFiNWY5MmMwMDI4NDI1YzA1YTQvKi5tM3U4P3Nlc3Npb25JRD0wYTM4NTlmNi0yNjY4LTQ0NDEtOTE4NS0wODZhZTZhZDY4ZmEiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE3MTY4MTg4ODR9fX1dfQ__&Signature=Xddda-xbZOiIehzAyAWnUzBhMwdLcsdsHUW3NnVHF4O272nh8-iHFI0VPqWw9-n4aTUHF00Ty45J60cKYNgdrRLncPtLnXoWkDMJUvFNstYu63tyDcJIWX5cyShBp1Gds~CgQ8Ns6NAVxn~mCsZ1Cxb4yK~p5AeLs0YArXmp~Gp07q99KWVmrxhTaaDHoTAz6UJznDHzVsErUk1FyC~ezsHJ-4Muy9mf9AEeeGG-m67gtrrgdocty8D5OR6PHq7mtORPOA594oE-WeOrxBMErZOHzz-ck9p4m-NpOWn8aLXNMssAsziVacRq-kTeweaLAB9vOwiFsyi9YUBb~y1p2A__&Key-Pair-Id=APKAIB5DGCGAQJ4GGIUQ&sessionID=0a3859f6-2668-4441-9185-086ae6ad68fa.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_cf0b6673-f855-45e8-ba7f-15ff62fda442",
    "downloadable_id": "downloadable_5843639359853216"
  }
}

**Comment by Mattmjs** [comment]
Looks like one of those non standard video streams like what was found here: https://github.com/aclap-dev/video-downloadhelper/discussions/162#discussioncomment-8534670

**Comment by geh2024** [comment]
It looks more like Fansly download where the actual video URL is hidden and this site sends a dummy video download. Unless it’s shown on an assign player which a video screen capture works. I don’t about a stream video capture?

 

Fansly Downloader has that work around but it’s not working for this site as it can’t find the video.
